pages/requests/create-replacement.tsx

```diff
@@ -0,0 +1,184 @@
+import Layout from '@components/admin/Layout'; // Layout component
+import { Text, Box, Flex, Stack, Button, GridItem, Input } from '@chakra-ui/react'; // Chakra UI
+import { useState } from 'react'; // React
+import PermitHolderInformationForm from '@components/admin/requests/forms/PermitHolderInformationForm'; //Permit holder information form
+import { PermitHolderInformation } from '@tools/components/admin/requests/forms/types'; //Permit holder information type
+import PaymentDetailsForm from '@components/admin/requests/forms/PaymentDetailsForm'; //Payment details form
+import { PaymentDetails } from '@tools/components/admin/requests/forms/types'; //Payment details type
+import { PaymentType, Province } from '@lib/graphql/types'; //GraphQL types
+import { ReasonForReplacement } from '@tools/components/admin/requests/forms/types';
+import { ReasonForReplacement as ReasonForReplacementEnum } from '@lib/graphql/types'; // Reason For Replacement Enum
+import ReasonForReplacementForm from '@components/admin/requests/forms/ReasonForReplacementForm';
+
+export default function CreateReplacement() {
+  const [permitHolderID] = useState('303240');
+  const [permitHolderInformation, setPermitHolderInformation] = useState<PermitHolderInformation>({
+    firstName: '',
+    lastName: '',
+    email: '',
+    phone: '',
+    addressLine1: '',
+    addressLine2: '',
+    city: '',
+    postalCode: '',
+  });
+
+  const [reasonDetails, setReason] = useState<ReasonForReplacement>({
+    reason: ReasonForReplacementEnum.Other,
+    lostTimestamp: null,
+    lostLocation: null,
+    description: null,
+    stolenJurisdiction: null,
+    stolenPoliceFileNumber: null,
+    stolenPoliceOfficerName: null,
+  });
+
+  const [paymentDetails, setPaymentDetails] = useState<PaymentDetails>({
+    paymentMethod: PaymentType.Mastercard,
+    donationAmount: 25,
+    shippingAddressSameAsHomeAddress: true,
+    shippingFullName: '',
+    shippingAddressLine1: '',
+    shippingAddressLine2: '',
+    shippingCity: '',
+    shippingProvince: Province.Bc,
+    shippingPostalCode: '',
+    billingAddressSameAsHomeAddress: true,
+    billingFullName: '',
+    billingAddressLine1: '',
+    billingAddressLine2: '',
+    billingCity: '',
+    billingProvince: Province.Bc,
+    billingPostalCode: '',
+  });
+
+  return (
+    <Layout footer={false}>
+      <GridItem display="flex" flexDirection="column" colSpan={12} paddingX="108px">
+        <Flex>
+          <Text textStyle="display-large">
+            {`New Replacement Request (User ID:`}
+            <Box as="span" color="primary">
+              {' '}
+              {permitHolderID}
+            </Box>
```

Comment:
Make this a Next.js `Link`, see feedback on https://github.com/uwblueprint/richmond-centre-for-disability/pull/101